bin/MadGraph5_aMCatNLO/cards/WZ_leptonic/WLLjj_dim8_ewk_run_card.dat

```diff
@@ -0,0 +1,269 @@
+#*********************************************************************
+# Minimum and maximum E's (in the center of mass frame)              *
+#*********************************************************************
+  0  = ej     ! minimum E for the jets 
+  0  = eb     ! minimum E for the b 
+  0  = ea     ! minimum E for the photons 
+  0  = el     ! minimum E for the charged leptons 
+ -1   = ejmax ! maximum E for the jets
+ -1   = ebmax ! maximum E for the b
+ -1   = eamax ! maximum E for the photons
+ -1   = elmax ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+  -1  = etaj    ! max rap for the jets 
+  -1  = etab    ! max rap for the b
+ 2.5  = etaa    ! max rap for the photons 
+  -1  = etal    ! max rap for the charged leptons 
+ 0.6  = etaonium ! max rap for the quarkonium states
+   0  = etajmin ! min rap for the jets
+   0  = etabmin ! min rap for the b
+   0  = etaamin ! min rap for the photons
+   0  = etalmin ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+ 0.0 = drjj    ! min distance between jets 
+ 0.0 = drbb    ! min distance between b's 
+ 0.01 = drll    ! min distance between leptons 
+ 0.0 = draa    ! min distance between gammas 
+ 0.0 = drbj    ! min distance between b and jet 
+ 0.0 = draj    ! min distance between gamma and jet 
+ 0.0 = drjl    ! min distance between jet and lepton 
+ 0.0 = drab    ! min distance between gamma and b 
+ 0.0 = drbl    ! min distance between b and lepton 
+ 0.0 = dral    ! min distance between gamma and lepton 
+ -1  = drjjmax ! max distance between jets
+ -1  = drbbmax ! max distance between b's
+ -1  = drllmax ! max distance between leptons
+ -1  = draamax ! max distance between gammas
+ -1  = drbjmax ! max distance between b and jet
+ -1  = drajmax ! max distance between gamma and jet
+ -1  = drjlmax ! max distance between jet and lepton
+ -1  = drabmax ! max distance between gamma and b
+ -1  = drblmax ! max distance between b and lepton
+ -1  = dralmax ! maxdistance between gamma and lepton
+#*********************************************************************
+# Minimum and maximum invariant mass for pairs                       *
+# WARNING: for four lepton final state mmll cut require to have      *
+#          different lepton masses for each flavor!                  *           
+#*********************************************************************
+ 100 = mmjj    ! min invariant mass of a jet pair 
+ 100 = mmbb    ! min invariant mass of a b pair 
+ 0   = mmaa    ! min invariant mass of gamma gamma pair
+ 60  = mmll    ! min invariant mass of l+l- (same flavour) lepton pair
```

Comment:
This will give no effect to your phase space.
The particles that are produced/decayed with madspin won't have such cuts applied.
Are you aware of this?

Reply:
@sihyunjeon 
Just to make sure. We don't have jets in the WW final state, just in the full process. Hence, the dijet cut mass can be there anyway. Right?
And regarding the mll cut, if we produce "Z" bosons, then they'll be on-shell and we don't need the mll cut. Can you please 
confirm?

Reply:
Sorry for not being so clear.

Yes when you do
MG : p p > X1 X2 X3  + MS : X1 > Y1 Y2
The cuts will be applied to X1 X2 X3 but not to Y1 Y2 which decays in MS.
So you are understanding is correct, due to onshell Z requirements mmll cut will not really be useful (your mll will be bound to Z mass window) and the effective cut will be dijet mass

Reply:
Hi @geet59 @SandeepHundal18, I agree with @sihyunjeon. I don't assume you intend to mix leptons from W and Z? If not please remove. The syntax above is already on-shell Z. Suggest to remove. Please let us know.